client/components/styles/MainNavDiv.js

```diff
@@ -10,7 +10,8 @@ export const MainNavDiv = styled.nav`
   color: white;
   background-color: black;
   border: none;
-  position: fixed;
+  position: -webkit-sticky;
+  position: sticky;
```

Comment:
`position: fixed` was hiding part of the pagination controls on the `/products` page, so I updated it to this, which should become fixed when users scroll down.

Reply:
Thanks for catching that!